maintenance_plan/models/maintenance_equipment.py

```diff
@@ -168,7 +168,7 @@ def _compute_next_maintenance(self):
                 )
                 .mapped("request_date")
             )
-            if len(next_plan_dates + next_unplanned_dates) <= 0:
+            if (not next_plan_dates) or (not any(next_plan_dates)) or len(next_plan_dates + next_unplanned_dates) <= 0:
```

Comment:
Pour quelle raison tu dois faire le not next_plan_dates et le not any(next_plan_dates) ?

Reply:
Hello Nadège,
en fait en testant le développement en v16 sur une db vide et nouvelle, j'ai eu une erreur à la simple création d'une ligne de maintenance plan, qui donnait à la fin next_plan_dates = [False]. C'est sûrement un raccourci fonctionnel que j'ai utilisé sans le faire exprès mais j'ai donc considéré ce cas de figure dans le premier if pour mettre "equipment.next_action_date" à None.

Tu voudrais que je le rectifie autrement ?
Merci.

@ngrandjean

Reply:
Ah ok je comprends :) Tu n'as pas de problèmes dans le else si next_plan_date est [False] ? (pour le min(next_plan_dates + next_unplanned_dates))
Sinon tu peux aussi changer le remplissage du tab next_plan_date pour ne pas ajouter "False" dedans quand les planifications de maintenance n'en ont pas

Reply:
Salut Nadège,
Non car si next_plan_dates = [False] alors ce que j'ai rajouté permet justement de l'englober dans le premier if, et le else ne sera pas parcouru, car ça répond à la condition "not any(next_plan_dates)".

Oui je peux aussi rajouter un filtre à ce niveau là à la place.

J'ai rajouté d'autres petites modifications qui donnaient des bugs que je n'avais pas vu.

@ngrandjean